_plugins/jekyll-generate-taxa.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
```

Comment:
Can we use [jekyll-datapage_gen](https://github.com/avillafiorita/jekyll-datapage_gen) instead? If not, this plugin should be included in the [legumeinfo-theme repo](https://github.com/legumeinfo/legumeinfo-jekyll-theme), not the site repo.

Reply:
Thanks for the review and the comments! I'll try to make the changes.
I am open to using jekyll-datapage_gen. I am not sure the best way to do that. Do you want to add that to the legumeinfo-theme repo?